### PR TITLE
MGMT-10868: Adding periodic for latebinding SNO day2 workers.

### PR DESCRIPTION
**Pull request by paul-maidment**
This commit introduces a periodic test for latebinding SNO day2 workers.
It was removed from https://github.com/openshift/release/pull/29787 in
order to allow that to be merged while
https://github.com/openshift/assisted-service/pull/4058 is tested.

### CODE REVIEW

File: ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-master-periodics.yaml [hunk 1]
```diff
@@ -663,6 +663,80 @@ periodics:
...
+  - base_ref: master
+    org: openshift
+    repo: assisted-service
+  labels:
+    ci-operator.openshift.io/cloud: packet-edge
+    ci-operator.openshift.io/cloud-cluster-profile: packet-assisted
+    ci-operator.openshift.io/variant: edge
+    ci.openshift.io/generator: prowgen
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-assisted-service-master-edge-e2e-metal-assisted-operator-ztp-sno-day2-workers-latbnd-prdc
```
Comment (omertuc) [inline ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-master-periodics.yaml:682]: Needs `reporter_config`

**Comment by omertuc** [comment]
https://github.com/openshift/assisted-service/pull/4065 is green we can move on with this

**Comment by omertuc** [comment]
/lgtm

Wait for [rehearsal](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/30075/rehearse-30075-periodic-ci-openshift-assisted-service-master-edge-e2e-metal-assisted-operator-ztp-sno-day2-workers-latbnd-prdc/1546525906795761664) before unholding

File: ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml [hunk 2]
```diff
@@ -508,7 +508,29 @@ tests:
...
+        EXTRA_WORKER_VCPU=8
+        EXTRA_WORKER_MEMORY=32768
+        EXTRA_WORKER_DISK=100
+        PROVISIONING_NETWORK_PROFILE=Disabled
+        REDFISH_EMULATOR_IGNORE_BOOT_DEVICE=True
+      DISCONNECTED: "false"
+      IP_STACK: v4
+      NUM_EXTRA_WORKERS: "2"
+    workflow: assisted-baremetal-operator-ztp
+- as: e2e-metal-assisted-operator-capi
```
Comment (osherdp) [inline ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml:533]: @paul-maidment why did you change capi job name?
or is it not on purpose?
